packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -349,22 +345,17 @@ export const messageThreadSelectorWithThread: () => MessageThreadSelector = () =
       };
     }
   );
-
 const sanitizedMessageContentType = (type: string): MessageContentType => {
   const lowerCaseType = type.toLowerCase();
   return lowerCaseType === 'text' || lowerCaseType === 'html' || lowerCaseType === 'richtext/html'
     ? lowerCaseType
     : 'unknown';
 };
-
 const isMessageValidToRender = (message: ChatMessageWithStatus): boolean => {
   if (message.deletedOn) {
     return false;
   }
-  if (
-    message.metadata?.['fileSharingMetadata'] ||
-    /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */ message.content?.attachments
-  ) {
+  if (message.metadata?.['fileSharingMetadata'] || message.content?.attachments) {
```

Comment:
Can we avoid direct access to objects via a string?
Is there a typing on metadata that can be used / extended?